src/Compatibility/Core/src/Android/AppCompat/Platform.cs

```diff
@@ -7,14 +7,15 @@
 using Android.OS;
 using Android.Views;
 using Android.Views.Animations;
+using Microsoft.Maui.Controls.Compatibility.Platform.Android.AppCompat;
 using Microsoft.Maui.Controls.Internals;
 using Microsoft.Maui.Controls.Platform;
 using Microsoft.Maui.Graphics;
 using AView = Android.Views.View;
 
-namespace Microsoft.Maui.Controls.Compatibility.Platform.Android.AppCompat
+namespace Microsoft.Maui.Controls.Compatibility.Platform.Android
 {
-	internal class Platform : BindableObject, IPlatformLayout, INavigation, IDisposable
+	public class Platform : BindableObject, IPlatformLayout, INavigation
```

Comment:
Why is this public now?

Reply:
Added a note on the description

> Condensed down Platform on Android to a single file. This unfortunately caused a bunch of renames. I removed the AppCompat one because the one in the non appcompat namespace has public APIs. Having the two was making the code frustrating to navigate

Reply:
Can we make it non-public? Is there anything useful on there that apps use?

Reply:
3rd party libraries use it to create renderers

It's really the only way to create renderers if you're inside your own renderer. So if someone has a control with templating or anything like that then they will use these APIs to create the renderers for those components